### PR TITLE
feat: include proportion of edges in aggregates metric

### PR DESCRIPTION
**Pull request by ambarrio**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This PR include a metric to track proportion of edges in cells with antibody outliers.

Fixes #EXE-1380

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Local tests

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## PR checklist:

- [x] This comment contains a description of changes (with reason).
- [x] My code follows the style guidelines of this project
- [x] Make sure your code lints, is well-formatted and type checks
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If you've added a new stage - have you followed the pipeline CLI conventions in the [USAGE guide](../USAGE.md)
- [ ] [README.md](./README.md) is updated
- [ ] If a new tool or package is included, update poetry.lock, [the conda recipe](../conda-recipe/pixelator/meta.yaml) and [cited it properly](../CITATIONS.md)
- [ ] Include any new [authors/contributors](../AUTHORS.md)
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and documentation and corrected any misspellings
- [ ] Usage Documentation is updated
- [x] If you are doing a [release](../RELEASING.md#Releasing), or a significant change to the code, update [CHANGELOG.md](../CHANGELOG.md)

### CODE REVIEW

File: src/pixelator/report/__init__.py [hunk 1]
```diff
@@ -551,7 +552,10 @@ def _collect_metrics(adata: AnnData) -> Dict[str, float]:
                 adata.obs["tau_type"]
             )
             metrics["reads_of_aggregates"] = adata[aggregates_mask].obs["reads"].sum()
-            metrics["umis_of_aggregates"] = adata[aggregates_mask].obs["umi"].sum()
+            metrics["edges_of_aggregates"] = adata[aggregates_mask].obs["edges"].sum()
```
Comment (johandahlberg) [inline src/pixelator/report/__init__.py:555]: Very small thing, but shouldn't in be `edge_in_aggregates`?

**Review by johandahlberg — APPROVED** [review]
Small comment, but it looks good to go for me.

File: src/pixelator/report/__init__.py (hunk 1 above)
Comment (ambarrio) [reply src/pixelator/report/__init__.py]: Good suggestion. I renamed several metrics and realized there was one left unused so removed that one too.

**Review by ptajvar — APPROVED** [review]
Looks good!

**Pull request closed without merging**